src/NzbDrone.Api/Calendar/CalendarFeedModule.cs

```diff
@@ -37,6 +38,7 @@ private Response GetCalendarFeed()
             var end = DateTime.Today.AddDays(futureDays);
             var unmonitored = false;
             var premiersOnly = false;
+			var allDay = false;
```

Comment:
Replace tabs with the appropriate number of spaces.
Advice: Install EditorConfig extension in your editor. It'll automatically adjust the editor settings based on `.editorconfig` files in the sources.